internal/metaquery/handler_inspect.go

```diff
@@ -68,6 +70,8 @@ func listViewSchema(ctx context.Context, input *HandlerInput, viewName string) e
 	return nil
 }
 
+// getPluginForTable returns the plugin name and version for a given table name.
+// note - this looks at the installed plugins and their version file entry, not only the version file
 func getPluginForTable(ctx context.Context, tableName string) (string, error) {
 	prefix := strings.Split(tableName, "_")[0]
```

Comment:
This might not work for entirely custom tables?

Reply:
I'll leave this for you to look at once this is in